src/pages/admin/student/edit/index.ts

```diff
@@ -0,0 +1,5 @@
+import { EditStudentPageContainer } from './pageContainer';
```

Comment:
Other way to export this file:

```typescript
export { EditStudentPageContainer } from './pageContainer';
```
or even

```typescript
export * from './pageContainer';
```

But I remember that we are using this way to export, so it's ok like now.